votor/src/voting_loop.rs

```diff
@@ -797,6 +799,7 @@ impl VotingLoop {
         vote_receiver: &VoteReceiver,
         cert_pool: &mut CertificatePool,
         commitment_sender: &Sender<AlpenglowCommitmentAggregationData>,
+        bls_sender: &Sender<BLSOp>,
```

Comment:
aplogies for the merge conflicts, after you rebase #279 we can add the bls sender here:
https://github.com/anza-xyz/alpenglow/blob/fbb25b3014fe9744cf79db9c6c82dc747e14ab6f/votor/src/certificate_pool_service.rs#L37
And then send the certificates that are returned here:
https://github.com/anza-xyz/alpenglow/blob/fbb25b3014fe9744cf79db9c6c82dc747e14ab6f/votor/src/certificate_pool_service.rs#L160